api/middleware.go

```diff
@@ -235,9 +235,15 @@ func RuntimeFromURLMiddleware(baseURL string) func(next http.Handler) http.Handl
 			// The first part of the path (after the version) determines the runtime.
 			// Recognize only whitelisted runtimes.
 			runtime := ""
-			switch { //nolint:gocritic // allow single-case switch for future expansions
+			switch {
```

Comment:
Should we also error check or throw?

Reply:
You mean in case no runtime matches? No, there are legitimate URL that just don't have an associated runtime, not even a runtime in the loose sense of "proper runtime or consensus". Example: `/v1/spec/v1.yaml`.